CONTENTS.md

```diff
@@ -0,0 +1,420 @@
+* Usage: gocd
+
+#### checknewer
+
+Checks if all packages in a repository are newer than all other repositories.
+
+* Sources: [checknewer.py](checknewer.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### deptool
+
+Assists in debugging dependencies
+
+* Sources: [deptool.py](deptool.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### requestfinder
+
+Allows to retrieve requests from OBS with quite elaborated queries.
+
+* Sources: [requestfinder.py](requestfinder.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### update_crawler (obsoleted by [origin-manager](#origin-manager))
+
+Create SRs for Leap.
+
+* Sources: [update_crawler.py](update_crawler.py) and [script](script).
+* Documentation: --
+* Package: openSUSE-release-tools-leaper
+* Usage: see [leaper](#leaper)
+
+#### create_staging
+
+Scripts and templates to create staging projects.
+
+* Sources: [staging_templates](staging_templates)
+* Documentation: --
+* Package: --
+* Usage: ?
+
+### Bots
+
+#### check_maintenance_incidents
+
+Handles maintenance incident requests
+
+* Sources: [check_maintenance_incidents.py](check_maintenance_incidents.py)
+* Documentation: [docs/maintbot.asciidoc](docs/maintbot.asciidoc)
+* Package: openSUSE-release-tools-maintenance
+* Usage: gocd
+
+#### leaper
+
+Implements Leap-style services for non-Factory projects (whatever that means).
+
+* Sources: [leaper.py](leaper.py)
+* Documentation: --
+* Package: openSUSE-release-tools-leaper
+* Usage: gocd
+
+#### origin-manager
+
+Keeps track of from what project a package originates, submit updates, review requests to detect origin changes, and enforce origin specific policies like adding appropriate reviews
+
+* Sources: [origin-manager.py](origin-manager.py) and [web](web)
+* Documentation: [docs/origin-manager.md](docs/origin-manager.md)
+* Package: openSUSE-release-tools-origin-manager
+* Usage: gocd
+
+#### staging-bot
+
+Assists in management of staging projects.
+
+* Sources: [devel-project.py][devel-project], [staging-report.py](staging-report.py), [suppkg_rebuild.py](suppkg_rebuild.py).
+* Documentation: --
+* Package: openSUSE-release-tools-staging-bot
+* Usage: gocd
+
+#### legal-auto
+
+Makes automatic legal reviews based on the legaldb API
+
+* Sources: [legal-auto.py](legal-auto.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### check_tags_in_requests
+
+Checks that a submit request has corrrect tags specified.
+
+* Sources: [check_tags_in_requests.py](check_tags_in_requests.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### abichecker
+
+Checks ABI compatibility in OBS requests.
+
+* Sources: [abichecker](abichecker)
+* Documentation: --
+* Package: openSUSE-release-tools-abichecker
+* Usage: gocd
+
+#### check_source_in_factory
+
+Checks if the sources of a submission are either in Factory or a request for Factory with the same
+sources exist.
```

Comment:
According to this https://github.com/openSUSE/openSUSE-release-tools/issues/1192 this tool is obsolete. It was kind of decided to drop it... but apparently it never happened since it's still mentioned at https://github.com/openSUSE/openSUSE-release-tools/blob/3990fecd037e4a08735b36a0d1cacf3cc5b7ac04/gocd/checkers.opensuse.gocd.yaml

Reply:
check_source_in_factory only scans Leap 15.0 - I think that's dead too :)